server/heroku/logs.go

```diff
@@ -20,13 +21,14 @@ type PostLogsForm struct {
 func (h *PostLogs) ServeHTTPContext(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
 	a, err := findApp(ctx, h)
 	if err != nil {
-		return err
+		return fmt.Errorf("error finding app: %v", err)
```

Comment:
Don't think we can do this here right now because of the way [we currently handle gorm.RecordNotFound](https://github.com/remind101/empire/blob/master/server/heroku/errors.go#L66-L68).